docs/proposals/karmada-components-extra-volumes/README.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Support to Specify Extra Volumes and Volume Mounts for Karmada Control Plane Components
+authors:
+- "@jabellard"
+reviewers:
+- "@RainbowMango"
+approvers:
+- "@RainbowMango"
+
+creation-date: 2024-07-29
+
+---
+
+# Support to Specify Extra Volumes and Volume Mounts for Karmada Control Plane Components
+
+
+## Summary
+
+This proposal aims to introduce the ability to specify extra volumes and volume mounts for Karmada control plane components when creating a Karmada instance managed by the Karmada operator. 
+This enhancement will extend the current configuration capabilities, allowing users to meet additional requirements such as having the ability to configure encryption at rest or to configure
+custom authentication webhooks.
+
+## Motivation
+Currently, the Karmada operator allows specifying extra arguments for the Karmada control plane components, but lacks support for specifying extra volumes and volume mounts for those components. 
+This limitation hinders certain use cases that require additional configurations for security and customization. For instance, as of today, it's not possible to configure a custom authentication 
```

Comment:
Could you please describe the use case in more detail? How this volumes will be used?